x/or/da/testutil/keeper.go

```diff
@@ -37,20 +37,16 @@ func DaKeeper(t testing.TB, encCfg simappparams.EncodingConfig) (keeper.Keeper,
 	accountKeeprMock := NewMockAccountKeeper(ctrl)
 	accountKeeprMock.EXPECT().GetParams(gomock.Any()).Return(authtypes.DefaultParams()).AnyTimes()
 	rollupKeeperMock := NewMockRollupKeeper(ctrl)
-	rollupKeeperMock.EXPECT().GetAllRollup(gomock.Any()).Return(
-		[]rolluptypes.Rollup{
-			{
-				RollupName:     "rollup1",
-				Creator:        "creator1",
-				L1ToL2GasRatio: 30,
-				PermissionedAddresses: rolluptypes.Sequencers{
-					Addresses: []string{"sequencer1"},
-				},
+	rollupKeeperMock.EXPECT().GetAllRollup(gomock.Any()).Return([]rutypes.Rollup{
+		{
+			RollupName:     "rollup1",
+			L1ToL2GasRatio: 100,
+			PermissionedAddresses: rutypes.Sequencers{
+				Addresses: []string{"addr1", "addr2"},
```

Comment:
Why were the values ​​of L1ToL2GasRatio and Addresses changed?

Reply:
The former one is just done that and latter one is adding one more address to cover more general test case.